DNN Platform/Providers/HtmlEditorProviders/RadEditorProvider/DotNetNuke.RadEditorProvider.dnn

```diff
@@ -1,6 +1,6 @@
 ﻿<dotnetnuke type="Package" version="5.0">
   <packages>
-    <package name="DotNetNuke.RadEditorProvider" type="Provider" version="09.01.00">
+    <package name="DotNetNuke.RadEditorProvider" type="Provider" version="09.04.02">
```

Comment:
Do these need to be set to 9.4.3, or are they updated automatically now?

Reply:
They will be updated after release from now on. This will happen in an upcoming PR and we'll make an automated task to do this in the future.